contents/blog/the-posthog-array-1-43-0.md

```diff
@@ -0,0 +1,115 @@
+---
+date: 2023-01-16
+title: "Array 1.43.0: Massive performance improvements!"
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+categories: ['Release notes', 'Product updates']
+featuredImage: ../images/blog/posthog-array-blog.png
+featuredImageType: full
+---
+
+Want to know more about what we're up to? Check out [our roadmap](/roadmap) to see what we're working on and what new beta features are available! You can also subscribe to [our Hogmail newsletter](/newsletter)!
+
+## Update guide for self-hosted users
+
+Any notes about async migrations, etc. go here.
+
+## PostHog 1.43.0 release notes
+
+**Release highlights:**
+
+- [New: Performance improvements](#new-performance-improvements)
+- [New: Role based access](#role-based-access)
+- [New: Better insight searching](#new-better-insight-searching)
+- [New: Feature flag variant overrides](#new-feature-flag-variant-overrides)
+- [New: Export recordings to file](#new-export-recordings-to-file)
+- [Improved: More experiment variants and improved flow](#improved-more-experiment-variants-and-improved-flow)
+- [Improved: Query cancelling improvements](#improved-query-cancelling-improvements)
+- [Improved: Recording playback controls](#improved-recording-playback-controls)
+- [Improved: Browsing recordings and creating playlists](#improved-browsing-recordings-and-creating-playlists)
+
+### New: Performance improvements
+
+@Karl to add a sentence or two here. 
```

Comment:
```suggestion
We don't want you to spend time waiting on machines so performance has had a renewed focus. Some improvements shipped this release:

- ClickHouse 22.8 support. This release also enables an important query optimization which will speed up some expensive queries up to 30%.
- Reworked dashboard cache refreshing logic - your dashboards should now be more up-to-date
- When updating dashboard filters we now may hit the cache for faster results
- Improved query cancellation when navigating away from trends or dashboards
- Reduced dashboard API response size. Existing `items` field is now deprecated and removed next release
- Property filters modal has received a speedup
- Home dashboards are no longer loaded twice
- When refreshing dashboards we now reload tiles in the same order as they're placed in visually
- When loading dashboards, we no longer load data from redis twice
- When updating dashboard filters, we now use cached data instead of always refreshing immediately
- Trends queries received an update which will reduce cross-shard traffic significantly and speed up queries
- Improved celery housekeeping tasks to require less ClickHouse resources. Run 0008 async migration to get full benefits of this.
- Many other small and large API speedups 

Last but not least, we now also have a [ClickHouse manual](/handbook/engineering/clickhouse) where we're gathering information we have learned about the database powering PostHog.
```